src/main/java/mate/academy/Main.java

```diff
@@ -10,12 +10,12 @@
 public class Main {
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = new User("bill", "qwerty");
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
```

Comment:
You can just leave return and dont initialize user

Reply:
Sorry but I misunderstood You. I just tried to provide more information in the message.